guides/common/assembly_managing-locations.adoc

```diff
@@ -1,11 +1,4 @@
-[id="Managing_Locations_{context}"]
-= Managing Locations
-
-Locations function similar to organizations: they provide a method to group resources and assign hosts.
-Organizations and locations have the following conceptual differences:
-
-* Locations are based on physical or geographical settings.
-* Locations have a hierarchical structure.
+include::modules/con_managing-locations.adoc[]
```

Comment:
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`